go/vt/vtorc/inst/instance_dao.go

```diff
@@ -429,6 +441,12 @@ Cleanup:
 				instance.GtidErrant, err = vitessmysql.Subtract(redactedExecutedGtidSet.String(), redactedPrimaryExecutedGtidSet.String())
 			}
 		}
+		// update the errant gtid map
+		go func() {
```

Comment:
Curious why this is done in a goroutine rather than an inline sequential routine?

Reply:
We run this code path of refreshing tablet data parallelly for all the tablets sometimes. In that case, I didn't want this function to be blocked. I think it is okay for them to spawn go-routines that sequentially update the map asynchornously from the function. This map is used only for the metric, so as long as the updates flow, its all okay (even if slightly delayed)

Reply:
I considered an alternate where instead of updating the map on this read, I would just read the information from the database on a time-tick. This approach has a slight advantage wherein it is faster in its updation.

Reply:
That makes sense. Thank you!